### DISCUSSION TITLE
Disable the "Clicking anywhere outside of an open panel will close that panel."

### DISCUSSION THREAD

**Discussion by NorseGaud** [tailwindlabs/headlessui#1462] [Help]
I need to keep the panel active indefinitely until the user clicks a specific button. I see the docs say "Clicking anywhere outside of an open panel will close that panel." and I'm wondering what a good way to disable this is. I use Vue if that helps.

**Comment by charlesoestergaard** [comment]
Please share your code

> I need to keep the panel active indefinitely until the user clicks a specific button. I see the docs say "Clicking anywhere outside of an open panel will close that panel." and I'm wondering what a good way to disable this is. I use Vue if that helps.

**Reply by dpschen** [reply]
@NorseGaud: are you referring here to the Dialog component?
Because I was just about to write a feature request for a property to disable close on click outside.

**Comment by lucasra1** [comment]
> I need to keep the panel active indefinitely until the user clicks a specific button. I see the docs say "Clicking anywhere outside of an open panel will close that panel." and I'm wondering what a good way to disable this is. I use Vue if that helps.

I think you could just omit the `Dialog.Panel` Component so the Dialog cant check for outsideClick behavior
